appcli/functions.py

```diff
@@ -51,6 +54,25 @@ def print_header(title):
         title.upper(),
     )
 
+def encrypt_text(cli_context, text):
```

Comment:
Moved `encrypt_helper` function from `appcli/appcli/commands/helpers.py` to `appcli/functions.py` and renamed function to `encrypt_text`